digitalstrategie/assets/scss/components/_ds-block-overwrites.scss

```diff
@@ -4,6 +4,24 @@
         max-width: 60%;
         margin: auto !important;
     }
+
+    // overwrite of herounit text box
+    .ds-hero-image__overlay {
+        margin-top: -18 * $spacer !important;
+        margin-bottom: 5 * $spacer !important;
+        text-align: center;
+    }
+
+    .ds-hero-image__copyright {
```

Comment:
caption does not appear below image on mobile screensize.
And for larger screensize the white background of the caption is quite big.

<img width="2938" alt="Bildschirmfoto 2022-03-21 um 10 50 21" src="https://user-images.githubusercontent.com/52801593/159246769-b5887dda-3da3-4a26-b547-06caa6fc1a17.png">

Reply:
Yo haven't seen that before, what browser and how wide is screen so I can replicate?

Reply:
using chrome and above 1660 px width it starts to "grow"